### DISCUSSION TITLE
Updates to the documentation

### DISCUSSION THREAD

**Discussion by dundurlunka** [rebus-org/Rebus#1077] [Ideas]
I've been updating the Rebus configurations in a .NET Framework service for the past several days and was stuck several times with different problems. In the end, everything went fine but I had to dig quite deep in order to find solutions to my problems, so I think these small additions to the documentation would be appreciated by future users:

- I think the `startAutomatically` parameter of `RegisterRebus` method, part of the Rebus.SimpleInjector, should be mentioned somewhere in the documentation of the Rebus.SimpleInjector. It's a very nice feature that I am very glad is there but I had to open the source code of the NuGet to find it.
- Speaking of `Rebus.SimpleInjector`, this line is written in its documentation: `container.RegisterCollection<IHandleMessages<SomeMessage>>(new []{ typeof(SomeMessageHandler) });`. However Visual Studio prompts me that this method is obsolete and suggests another approach which works fine.

Sorry if this is not the correct place for such suggestions.

Thank you.

**Comment by mookid8000** [comment]
I've updated [the readme](https://github.com/rebus-org/Rebus.SimpleInjector/blob/master/README.md) now as per your suggestions, and I've also added a bit about Rebus.SimpleInjector's handler registration methods. 🙂 

Thanks for letting me know the docs were outdated.

**Reply by dundurlunka** [reply]
Thank you for the quick response :)
